source/infrastructure/mod-ood-proxy/installation.rst

```diff
@@ -5,31 +5,37 @@ Installation
 
 #. Download the latest stable release and unpack it:
 
-   .. code-block:: sh
+   Download the latest release as of this writing:
+   
+   .. code-block:: console
 
-      # Set latest release as of writing this
-      LATEST="{{ mod_ood_proxy_version }}"
+      $ LATEST="{{ mod_ood_proxy_version }}"
+      $ wget --content-disposition https://github.com/OSC/mod_ood_proxy/archive/v${LATEST}.tar.gz
 
-      # Download latest release
-      wget --content-disposition https://github.com/OSC/mod_ood_proxy/archive/v${LATEST}.tar.gz
+   Unpack archive:
```

Comment:
In fact remove all these original comment lines and pack it into a single code-block like before.